pom.xml

```diff
@@ -40,23 +41,23 @@ THE SOFTWARE.
 
   <name>Maven Integration plugin</name>
   <description>This plug-in provides, for better and for worse, a deep integration of Jenkins and Maven: Automatic triggers between projects depending on SNAPSHOTs, automated configuration of various Jenkins publishers (Junit, ...).</description>
-  <url>https://github.com/jenkinsci/maven-plugin</url>
+  <url>https://github.com/jenkinsci/${project.artifactId}</url>
 
   <properties>
     <revision>3.16</revision>
     <changelist>-SNAPSHOT</changelist>
-    <gitHubRepo>jenkinsci/maven-plugin</gitHubRepo>
-    <jenkins.version>2.289.1</jenkins.version>
+    <gitHubRepo>jenkinsci/${project.artifactId}</gitHubRepo>
+    <jenkins.version>2.321</jenkins.version>
```

Comment:
It's to get the new version of Guava/Guice and avoid having to embed them. Since there's no urgent need to merge and release this, let's wait until the next LTS.